pyEOS/eos.py

```diff
@@ -157,17 +157,29 @@ def load_candidate_config(self, filename=None, config=None):
         else:
             self.candidate_config.load_config(config=config)
 
-    def compare_config(self):
+    def compare_replace_config(self):
```

Comment:
But that´s why we have napalm as another project. To unify. pyEOS, pyIOSXR, pyEZ and pyFG and others that might come should do whatever they have to do to make our lives easier to interact with their APIs without having to go deep down to netconf, expect or the eAPI. Then napalm is what we do to abstract the details and provide the same user experience.